config/production.json

```diff
@@ -8,7 +8,9 @@
         "acquire": 60000
       },
       "dialectOptions": {
-        "ssl": true
+        "ssl": {
+          "rejectUnauthorized": true
```

Comment:
If we set it to `true` we'll need to supply the certificate, otherwise, the connection won't work.
If we want to explicitly disable, this should be set to false.

Reply:
Good catch @kewitz ! I looked at it many times and missed it.

Reply:
Yes, this argument is passed to `pg` which passes to Node's `tls` library, you can check it here:
https://nodejs.org/api/tls.html#tls_tls_connect_options_callback
Notice that our Postgres host is `amazonaws.com` if you're going after that certificate.